### PR TITLE
Manager UI update progress + ui polish

### PR DESCRIPTION
**Pull request by LFBarreto**
**Manager V2:**
- Update progress bar added
- Device storage bar graph updated with some hover animation
- Manager connect center positioning issue fixed
- View split and memoized to avoid too many rerenders

**UI polishes**
- FadeInOut component added
- Checkbox border color contrast upped a bit
- Modal scroll bar now invisible 

### Type

UI Polish...

### Parts of the app affected / Test plan

Manager V2

### CODE REVIEW

File: src/renderer/components/ManagerConnect/index.js [hunk 1]
```diff
@@ -116,6 +115,7 @@ const Wrapper = styled.div`
   flex: 1;
   align-items: center;
   justify-content: center;
+  padding-bottom: ${p => p.theme.space[6] + p.theme.sizes.topBarHeight};
```
Comment (gre) [inline src/renderer/components/ManagerConnect/index.js]: beware we use this component in onboarding, in genuine check modal. we don't want this padding here i feel.

File: src/renderer/components/ManagerConnect/index.js [hunk 2]
```diff
@@ -142,9 +142,12 @@ const Footer = styled.div`
   align-items: center;
 `;
 
+const TroobleshootingWrapper = styled.div`
+  margin-top: auto;
+`;
+
 const ManagerConnect = ({
   reduxDevice,
-  Success,
```
Comment (gre) [inline src/renderer/components/ManagerConnect/index.js:146]: oops

File: src/renderer/screens/manager/DeviceStorage/index.js [hunk 3]
```diff
@@ -255,4 +266,4 @@ const DeviceStorage = ({ state, deviceInfo }: *) => {
   );
 };
 
-export default DeviceStorage;
+export default memo<Props>(DeviceStorage, isEqual);
```
Comment (gre) [inline src/renderer/screens/manager/DeviceStorage/index.js]: I don't think it's a good idea to use the (deep) isEqual of lodash here. especially that it's going to apply each time on the state object! which is often changing anyway, we should probably instead split the component in more pieces

Comment (LFBarreto) [reply src/renderer/screens/manager/DeviceStorage/index.js]: This one was unnecessary sorry :bowing_man: 
I replaced the deep Equal with a more tailored solution on each item lines that's where the re-render bottleneck is concentrated when progress is ongoing :+1:

**Review by gre — APPROVED** [review]

**Review by valpinkman — APPROVED** [review]

**Review by Arnaud97234 — APPROVED** [review]

**Pull request merged**